casper/src/main/resources/PoS.rhox

```diff
@@ -330,7 +331,7 @@ new  PoS,
                   getBlockData!(*blockDataCh) |
 
                   runMVar!(*stateCh, *mvarProcessCh, *ackCh) |
-                  for(_, @blockNumber, _ <- blockDataCh;
+                  for(@blockNumber, _, _ <- blockDataCh;
```

Comment:
This is fix unrelated to this PR. Block number is the first parameter. [SystemProcesses.scala](https://github.com/tgrospic/rchain/blob/5407c0047/rholang/src/main/scala/coop/rchain/rholang/interpreter/SystemProcesses.scala#L250)
Thanks @Isaac-DeFrain!